### PR TITLE
Hosting startup updates

### LINKED ISSUE #11575: Activation from a runtime store-deployed assembly
It is realy hard to follow in this section! what stop 1 and step 2 mean!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 613ad64e-bc62-edf7-2e86-42210de718f4
* Version Independent ID: aabbbc32-3237-5ef9-c422-892dc6884a22
* Content: [Use hosting startup assemblies in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/platform-specific-configuration?view=aspnetcore-2.2#feedback)
* Content Source: [aspnetcore/fundamentals/host/platform-specific-configuration.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/host/platform-specific-configuration.md)
* Product: **aspnet-core**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #11575 

Let's flesh out the script remarks in the topic. Script remarks aren't localized. These are the updates that I couldn't easily make from the phone at the conference (no laptop!). I did two PRs from the phone ... they were ⚔️ *hand-to-hand combat* ⚔️ to do on a phone. :smile:

### CODE REVIEW

**Comment by pakrym** [comment]
cc @anurse @Tratcher as I'm moving away from owning HostingStartup

**Comment by guardrex** [comment]
I left the comments in the scripts. Compare the text updates to ...

*build.ps1*

https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/host/platform-specific-configuration/samples/2.x/RuntimeStore/build.ps1

*deploy.ps1*

https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/host/platform-specific-configuration/samples/2.x/RuntimeStore/deploy.ps1
